cli/command/service/ps.go

```diff
@@ -58,8 +59,11 @@ func runPS(dockerCli *command.DockerCli, opts psOptions) error {
 	serviceIDFilter := filters.NewArgs()
 	serviceNameFilter := filters.NewArgs()
 	for _, service := range opts.services {
+		// default to container runtime
 		serviceIDFilter.Add("id", service)
+		serviceIDFilter.Add("runtime", string(swarmtypes.RuntimeContainer))
```

Comment:
These are a bit different.  This cannot be called from the API as these are intended to be used internally for filtering.  Should we doc these now or wait until we support the custom filtering based upon runtime type?